README.md

```diff
@@ -32,6 +32,8 @@ pip3 uninstall navalmartin_mir_db_utils
 
 ## How to use
 
+Create a session
```

Comment:
create a session and load `environmental` variables, perhaps?

Reply:
You cannot create a session without the MONGODB_URL so I left this to the application side how this will be done